hack/gen-swagger-doc/run-gen-swagger-docs.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
```

Comment:
Does this script run fine on release branch now? (Ref https://github.com/kubernetes/kubernetes/issues/18055)